tests/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(test_srcs test_bacio.pf)
+
+add_pfunit_ctest (bacio_test
+  TEST_SOURCES test_bacio_mod.pf
+  LINK_LIBRARIES bacio_4_static
```

Comment:
should'nt this be 
```
bacio::bacio_4_static
```

Reply:
I don't think so because the targets aren't imported here, they're just the local targets in the project. Only when you run `find_package` does the library gain the namespace, or so that was my thought. And it builds and runs.

Reply:
yes. you are right.